content/en/docs/reference/commands/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Command Line Reference"
-linkTitle: "Command Line Reference"
+title: "CLI"
```

Comment:
I think spelling it out is more appropriate. @jha-cloudbees wdyt?

Reply:
I am pro to **CLI**, it is far more common in doc sites.

Reply:
I prefer spelling it out once, then putting the abbreviation in parentheses after `(CLI)` then referring to it by abbreviation from then on.

Reply:
That's also what I "learned" :-) But maybe I am primed by working too long with RedHat docs people ;-)

Reply:
Don't you both come from Redhat come to think about it? :)

How about the left nav has it as **JX CLI** since that is crisp clear?

Just going by what by samples of this:

https://kubernetes.io/docs/reference/kubectl/overview/ **(kubectl CLI)**
https://docs.docker.com/reference/ucp/3.2/cli/ **( UCP CLI )**
https://www.pulumi.com/docs/reference/cli/ **(Pulumi CLI )**

@hferentschik + @jha-cloudbees